wwwroot/src/pages/WhyUs.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { Landing2 as WhyUs3 } from './Landing.js';
+
+import Layout from '../components/Layout';
+import { makeStyles } from '@material-ui/core/styles';
+
+import bannerImg from '../img/why-us/banner.jpg';
+import gettingImg from '../img/why-us/getting-better.jpg';
```

Comment:
Please move all the images into /public folder. Webpack is not configured to grab them from /img folder.

Reply:
Is it okay to make subfolders inside of /public to organize the images?

Reply:
added the images to /public